index.html

```diff
@@ -23,6 +23,13 @@
   <script src="node_modules/codemirror/mode/javascript/javascript.js" type="text/javascript"></script>
   <script src="node_modules/codemirror/addon/edit/matchbrackets.js" type="text/javascript"></script>
 
+  <!-- Enable CodeMirror's search addons -->
+  <script src="node_modules/codemirror/addon/search/search.js" type="text/javascript"></script>
+  <script src="node_modules/codemirror/addon/search/jumpToLine.js" type="text/javascript"></script>
```

Comment:
That is strange. You are right about that import, but it worked for me initially and fixing it didn't seem to change anything. So I'm not sure what the deal with that is.